packages/cms/README.md

```diff
@@ -37,3 +37,40 @@ import {Profile} from "@datawheel/canon-cms";
 |variable|description|default|
 |---|---|---|
 |`CANON_CMS_ENABLE`|Setting this env var to `true` allows access to the cms in production builds.|`false`|
+
+---
+
+## Migration
+
+For upgrading to new versions, there are currently three migration scripts:
+
+1) `npx migrate-legacy` (for DataUSA) 
+2) `npx migrate-0.1` (for CDC or other 0.1 CMS users)
+3) `npx migrate-0.6` (for 0.6 CMS users)
```

Comment:
@jhmullen those aren't the names of the scripts 😑 

(missing the `canon-cms-` prefix)

Reply:
Good catch, My bad, fixed.